distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java

```diff
@@ -692,21 +695,25 @@ private static Integer findPortNumber(Integer portToTry) {
               getClass().getResource("/etc/forms/imagery.xml"), "/etc/forms/imagery.xml"),
           installStartupFile(
               getClass().getResource("/etc/forms/contact-name.xml"), "/etc/forms/contact-name.xml"),
-          installStartupFile(
-              String.format(
-                  "priority \"grant\"; grant {permission java.io.FilePermission \"%s%s-\", \"read, write\"; };",
-                  new File("target" + File.separator + "solr")
-                      .getAbsolutePath()
-                      .replace("/", "${/}")
-                      .replace("\\", "${/}"),
-                  "${/}"),
-              "/security/itests-solr.policy"));
+          getFilePermissionsOption());
     } catch (IOException e) {
       LoggingUtils.failWithThrowableStacktrace(e, "Failed to deploy configuration files: ");
     }
     return new Option[0];
   }
 
+  protected Option getFilePermissionsOption() throws IOException {
+    return installStartupFile(
+        String.format(
+            FILE_PERMISSIONS,
+            new File("target" + File.separator + "solr")
```

Comment:
Won't the solr directory actually end up under `target/exam/<uuid>/solr` and not `target/solr`?

Reply:
Just answered my own question by looking a few lines down 😄